Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSource.java

```diff
@@ -139,6 +142,9 @@ private String getFaceImageUrl(Proxy proxy, CardDownloadData card, boolean isTok
 
         String apiUrl = ScryfallImageSupportCards.findDirectDownloadLink(card.getSet(), card.getName(), card.getCollectorId());
         if (apiUrl != null) {
+            if (apiUrl.endsWith("*/")) apiUrl= apiUrl.substring(0 , apiUrl.length() -2) +"★/" ;
```

Comment:
style:
- braces for all conditionals (line break after opening brace and before closing brace)
- spaces on both sides of  `= `, `+`, `-` operators